src/pages/certPinning/certPinning.ts

```diff
@@ -5,12 +5,14 @@ import { ToastController } from 'ionic-angular';
 
 @Component({
     selector: 'page-certPinning',
-    templateUrl: 'certPinning.html'
```

Comment:
@ciaranRoche now you have a number of unused html templates. We might want to remove them or create a fully-fledged underConstruction page with selector and place it directly into the html templates (if we want to leave them), by providing the selector in the html templates:

<page-underConstruction></page-underConstruction>